nvi-evaluator/src/main/java/no/sikt/nva/nvi/evaluator/calculator/CandidateCalculator.java

```diff
@@ -174,6 +179,16 @@ private List<Creator> extractVerifiedCreator(Publication publication) {
                    .toList();
     }
 
+    private Organization fetchOrganization(URI organizationId) {
+        var response = getResponse(organizationId);
+        if (isHttpOk(response)) {
+            return toCristinOrganization(response.body());
+        } else {
+            LOGGER.error(ERROR_COULD_NOT_FETCH_CRISTIN_ORG, organizationId, response.statusCode());
+            throw new RuntimeException(COULD_NOT_FETCH_CRISTIN_ORG_MESSAGE + organizationId);
```

Comment:
Do we need to it? maybe put it exception message?

Reply:
Want to keep the logging to be able to debug status code from cristin api